### ISSUE TITLE
Usability Testing Prep: Comparison Tool School Ratings -- Accessibility Feedback

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12780]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements

## Consider

* Additional round(s) of testing when code is relatively stable in Staging or other coded environment. Users who identify with cognitive issues, or who use assistive technology will benefit if they are included in research sessions.
* Mobile research when code is relatively stable.
* Possible third message for those cases where some users have rated an institution, but there are not enough ratings to show on the page. It's an edge case, but could help users understand the nuance and thought going into ratings.
* I'm interested to hear how the final scores are determined, if weights are added for things like time since rating, importance to the veteran community, etc. This is not an actionable item per se, but an opportunity to tailor the user experience.
  * If an algorithm is applied to weight or shape the rating scores, adding a short link to how it's calculated will go a long way with users. Many may not be interested, but those that are will take a lot from it, and may offer feedback on ways to smooth or adjust it. All of which feels like wins in the name of serving our users.

## Definition of Done

* Comments are acknowledged by the design team or PM
* Team comments inline or adds feedback if there are items they'd like to discuss further

**Comment by amyknox** [comment]
Thanks for the input, Trevor. We'll keep in mind the issue if someone rates a school but the display is delayed because there's insufficient ratings to display.  Still exploring what the threshold might be.  

We'll definitely be looking forward to testing this on staging, and with mobile users, to make sure the experience works across all platforms.
